### PR TITLE
Fix minor HTML issues in ReadFilter documentation

### PR DESCRIPTION
**Pull request by magicDGS**

### CODE REVIEW

**Comment by magicDGS** [comment]
@vdauwera and/or @sooheelee - Because you are the ones reviewing the documentation-related PRs, do you mind to have a look to this?

File: src/main/java/org/broadinstitute/hellbender/engine/filters/AlignmentAgreesWithHeaderReadFilter.java [hunk 1]
```diff
@@ -9,10 +9,13 @@
...
 /**
- * Filter out reads where the alignment does not match the contents of the header:
+ * Filter out reads where the alignment does not match the contents of the header.
+ *
+ * <p>The read does not match the contents of the header if:</p>
+ *
  * <ul>
- *     <li>Read aligned to non-existent contig</li>
- *     <li>Read aligned to a point after the end of the contig</li>
+ *     <li>It is aligned to non-existent contig</li>
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/engine/filters/AlignmentAgreesWithHeaderReadFilter.java]: to *a* non-existing contig

File: src/main/java/org/broadinstitute/hellbender/engine/filters/FragmentLengthReadFilter.java [hunk 2]
```diff
@@ -9,7 +9,8 @@
 
 /**
  * Keep only read pairs (0x1) with absolute insert length less than or equal to the given value.
- * Taking absolute values allows inclusion of pairs where the considered read's mate is at a smaller genomic coordinate.
+ *
+ * <p>Taking absolute values allows inclusion of pairs where the considered read's mate is at a smaller genomic coordinate.
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/engine/filters/FragmentLengthReadFilter.java]: where the mate of the read being considered is

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java [hunk 3]
```diff
@@ -36,7 +36,10 @@ private ReadFilterLibrary(){ /*no instance*/ }
             return read.isFirstOfPair();}}
 
     /**
-     * Keep only reads containing good CIGAR strings:
+     * Keep only reads containing good CIGAR strings.
+     *
+     * <p>Good CIGAR strings are:</p>
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java]: are -> have the following properties:

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java [hunk 4]
```diff
@@ -102,7 +107,7 @@ private ReadFilterLibrary(){ /*no instance*/ }
     /**
      * Keep only reads that have a mate that maps to the same contig (RNEXT is "="), is single ended (not 0x1) or has an unmapped mate (0x8).
      *
-     * See MappedReadFilter for criteria defining an unmapped read.
+     * <p>See MappedReadFilter for criteria defining an unmapped read.
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java]: missing closing tag

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java [hunk 5]
```diff
@@ -116,7 +121,7 @@ private ReadFilterLibrary(){ /*no instance*/ }
      * For paired reads (0x1), keep only reads that are mapped, have a mate that is mapped (read is not 0x8), and both
      * the read and its mate are on different strands (when read is 0x20, it is not 0x10), as is the typical case.
      *
-     * See MappedReadFilter for criteria defining an mapped read.
+     * <p>See MappedReadFilter for criteria defining an mapped read.
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java]: missing closing tag

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java [hunk 6]
```diff
@@ -226,7 +231,8 @@ private ReadFilterLibrary(){ /*no instance*/ }
 
     /**
      * Keep only reads with a valid alignment start (POS larger than 0) or is unmapped.
-     * See MappedReadFilter for criteria defining an unmapped read.
+     *
+     * <p>See MappedReadFilter for criteria defining an unmapped read.
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java]: missing closing tag

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java [hunk 7]
```diff
@@ -236,7 +242,9 @@ private ReadFilterLibrary(){ /*no instance*/ }
 
     /**
      * Keep only reads where the read end corresponds to a proper alignment -- that is, the read ends after the start
-     * (non-negative number of bases in the reference), calculated as:
+     * (non-negative number of bases in the reference).
+     *
+     * <p>>This is calculated as:</p>
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java]: extra bracket in first tag

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadGroupBlackListReadFilter.java [hunk 8]
```diff
@@ -18,8 +18,9 @@
 
 /**
  * Keep records not matching the read group tag and exact match string.
- * For example, this filter value:
- *   PU:1000G-mpimg-080821-1_1
+ *
+ * <p>For example, this filter value:
+ *   <code>PU:1000G-mpimg-080821-1_1</code>
  * would filter out a read with the read group PU:1000G-mpimg-080821-1_1
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/engine/filters/ReadGroupBlackListReadFilter.java]: missing closing tag

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadGroupReadFilter.java [hunk 9]
```diff
@@ -9,7 +9,8 @@
 
 /**
  * Keep only reads from the specified read group. Discards reads lacking an RG tag.
- * Matching is done by case-sensitive exact match.
+ *
+ * <p>Matching is done by case-sensitive exact match.
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/engine/filters/ReadGroupReadFilter.java]: missing closing tag

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadNameReadFilter.java [hunk 10]
```diff
@@ -9,7 +9,8 @@
 
 /**
  * Keep only reads with this read name.
- * Matching is done by case-sensitive exact match.
+ *
+ * <p>Matching is done by case-sensitive exact match.
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/engine/filters/ReadNameReadFilter.java]: missing closing tag

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadStrandFilter.java [hunk 11]
```diff
@@ -6,7 +6,9 @@
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
 /**
- * Keep only reads whose strand is either forward (not 0x10) or reverse (0x10), as specified. By default the filter keeps only forward reads (not 0x10).
+ * Keep only reads whose strand is either forward (not 0x10) or reverse (0x10), as specified.
+ *
+ * <p>By default the filter keeps only forward reads (not 0x10).
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/engine/filters/ReadStrandFilter.java]: missing closing tag

File: src/main/java/org/broadinstitute/hellbender/engine/filters/SampleReadFilter.java [hunk 12]
```diff
@@ -13,7 +13,8 @@
 
 /**
  * Keep only reads for a given sample.
- * Matching is done by case-sensitive exact match.
+ *
+ * <p>Matching is done by case-sensitive exact match.
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/engine/filters/SampleReadFilter.java]: missing closing tag

**Comment by vdauwera** [comment]
Done, thanks for the cleanup

File: src/main/java/org/broadinstitute/hellbender/engine/filters/FragmentLengthReadFilter.java (hunk 2 above)
Comment (magicDGS) [reply src/main/java/org/broadinstitute/hellbender/engine/filters/FragmentLengthReadFilter.java]: Done.

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java (hunk 3 above)
Comment (magicDGS) [reply src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java]: Done.

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java (hunk 4 above)
Comment (magicDGS) [reply src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java]: Done.

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java (hunk 5 above)
Comment (magicDGS) [reply src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java]: Done.

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java (hunk 6 above)
Comment (magicDGS) [reply src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java]: Done.

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java (hunk 7 above)
Comment (magicDGS) [reply src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java]: Removed.

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadGroupBlackListReadFilter.java (hunk 8 above)
Comment (magicDGS) [reply src/main/java/org/broadinstitute/hellbender/engine/filters/ReadGroupBlackListReadFilter.java]: Done.

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadGroupReadFilter.java (hunk 9 above)
Comment (magicDGS) [reply src/main/java/org/broadinstitute/hellbender/engine/filters/ReadGroupReadFilter.java]: Done.

File: src/main/java/org/broadinstitute/hellbender/engine/filters/ReadStrandFilter.java (hunk 11 above)
Comment (magicDGS) [reply src/main/java/org/broadinstitute/hellbender/engine/filters/ReadStrandFilter.java]: Done.

File: src/main/java/org/broadinstitute/hellbender/engine/filters/SampleReadFilter.java (hunk 12 above)
Comment (magicDGS) [reply src/main/java/org/broadinstitute/hellbender/engine/filters/SampleReadFilter.java]: Done.

**Comment by magicDGS** [comment]
I added the missing closing tags and addressed the rest of the comments. Back to you @vdauwera!

**Comment by vdauwera** [comment]
looks good, thanks!

**Comment by magicDGS** [comment]
Thank you!

**Pull request merged**